test/specs/unit/utils/format-number.spec.js

```diff
@@ -8,26 +8,26 @@ describe('formatNumber()', () => {
   });
 
   it('should add a separator after 1st digit for numbers whose length modulo 3 is 1', () => {
-    expect(formatNumber(4552)).toMatch(/^4\./);
-    expect(formatNumber(5436542)).toMatch(/^5\./);
-    expect(formatNumber(4536543843049314)).toMatch(/^4\./);
+    expect(formatNumber(4552)).toMatch(/^4\,/);
```

Comment:
I think these tests should be removed or moved to another type of tests where we can set the locale.